test/integration/Main.hs

```diff
@@ -162,6 +162,7 @@ testScenarioSolution _ci _em =
         , testSolution (Sec 5) "Challenges/2048"
         , testSolution (Sec 3) "Challenges/word-search"
         , testSolution (Sec 3) "Challenges/ice-cream"
+        , testSolution (Sec 90) "Challenges/gopher"
```

Comment:
As is, this test take 45 seconds on my computer to complete.  So I'm not sure whether we actually want this to be part of the test suite.

Reply:
I have now fixed this by adding a `wait 64` in the "polling" loop of the built robots.